mesh/mesh.cpp

```diff
@@ -15172,6 +15284,18 @@ Mesh *Extrude2D(Mesh *mesh, const int nz, const real_t sz)
    return mesh3d;
 }
 
+bool Mesh::Conforming() const
+{
+   if (NURBSext)
+   {
+      return true;
```

Comment:
Should we add a comment here to indicate that patch conformity can be checked with `NURBSext::Nonconforming()`?